composer.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "josegonzalez/cakephp-version",
-    "description": "CakePHP plugin to allow versioning of records",
-    "type": "cakephp-plugin",
-    "keywords": ["cakephp", "revision", "version"],
+    "description": "Cake orm behavior to allow versioning of records",
+    "type": "library",
+    "keywords": ["cake", "orm", "revision", "version"],
```

Comment:
I may be wrong, I thought cakephp seems more like a framework. May be add cakephp also .